src/styles/index.scss

```diff
@@ -1,3 +1,74 @@
+$sell-size: 100px;
+$gap-size: 1px;
+$padding-size: 10px;
+$border-size: 1px;
+
+$calendar-width: (7 * $sell-size) + (6 * $gap-size);
+
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
 body {
   margin: 0;
 }
+
+.calendar {
+  padding: 10px;
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $calendar-width;
+
+  &__day {
+    position: relative;
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.5s;
+
+    &:first-child {
+      margin-left: (6 * $sell-size) + (6 * $gap-size);
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: #FFBFCB;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
+
+@each $day, $offset in $days {
+  .calendar--start-day-#{$day} .calendar__day:first-child {
+    margin-left: ($offset * $sell-size) + ($gap-size * $offset);
```

Comment:
Ensure that '$sell-size' is corrected to '$cell-size' for consistency and clarity in the code.